languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10026,6 +10026,35 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rulegroup id="QUELLE_OU_QU_ELLE" name="quelle ou qu'elle" default="temp_off">
+          <rule>
+              <pattern>
+                  <token postag="SENT_START" postag_regexp="yes"></token>
+                  <marker>
+                      <token regexp="yes">Qu(')?</token>
+                      <token regexp="yes">elle</token>
+                  </marker>
+                  <token postag="V.*" postag_regexp="yes" skip="-1"><exception postag="V.* sub" postag_regexp="yes"></exception></token>
+                  <token postag="M fin inte" postag_regexp="yes"></token>
```

Comment:
If your postag doesn't contain regex, no need to write `postag_regexp="yes"`

Reply:
If the string is empty, you can write `<token postag="M fin inte"/>` instead of `<token postag="M fin inte"></token>`